apps/next_dev_portfolio/components/Skills.js

```diff
@@ -0,0 +1,64 @@
+import { DotSmall } from "./Dot";
+import {
+  ReactIcon,
+  NextIcon,
+  TailwindIcon,
+  MUIIcon,
+  DockerIcon,
+  DjangoIcon,
+} from "./Icons/Devicons";
+import Link from "next/link";
+
+export default function Skills() {
+  const [width, height] = [100, 100];
+  return (
+    <>
+      <div id="skills"></div>
+      <section className="max-w-4xl px-4 mx-auto mt-32 space-y-8 text-center lg:mt-52">
+        <h2 className="text-4xl text-white">
+          I work with
+          <DotSmall />
+        </h2>
+
+        <div className="grid grid-cols-2 gap-4 text-white lg:grid-cols-6 ">
+          <p className="flex flex-col items-center justify-center">
```

Comment:
These links are repeated quite a lot, probably you can simply render it from an array of objects so that if you change something, you don't have to change another N item.

Reply:
I need to think a little bit more how to handle it. For sure I need add more icons (like other frameworks) and pin to it official websites to make this template more flexible.